addons/l10n_au/__openerp__.py

```diff
@@ -0,0 +1,38 @@
+# -*- encoding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+# Copyright (C) 2015 Willow IT Pty Ltd (<http://www.willowit.com.au>).
+
+{
+    'name': 'Australian - Accounting',
+    'version': '1.1',
+    'category': 'Localization/Account Charts',
+    'description': """
+Australian Accounting Module
+============================
+
+Australian accounting basic charts and localizations.
+
+Also:
+    - activates a number of regional currencies.
+    - sets up Australian taxes.
+    """,
+    'author': 'Richard deMeester - Willow IT',
+    'website': 'http://www.willowit.com',
+    'depends': ['base',
+                'account',
+                ],
+    'data': [
+             'data/account_chart_template.xml',
+             'data/account.account.template.csv',
+             'data/account.account.tag.csv',
+             'data/account.tax.template.csv',
+             'data/account_fiscal_position_tax_template.xml',
+             'data/account_chart_template.yml',
+             'data/localisation.xml',
+             'security/ir.model.access.csv',
```

Comment:
I will remove - I was copying the fact that many of the other base charts seemed to have a security reference to empty files.